src/Bridge/Symfony/Bundle/Resources/config/hydra.xml

```diff
@@ -22,6 +22,7 @@
 
         <service id="api_platform.hydra.listener.response.add_link_header" class="ApiPlatform\Core\Hydra\EventListener\AddLinkHeaderListener">
```

Comment:
It would be cleaner to remove the service if this flag is set.

Reply:
Yup, but wouldn't this be in opposition to the fact that hydra *needs* this header ? It's the same thing as what I proposed, but still...